src/presentation/http/router/noteSettings.ts

```diff
@@ -117,6 +117,19 @@ const NoteSettingsRouter: FastifyPluginCallback<NoteSettingsRouterOptions> = (fa
           $ref: 'NoteSchema#/properties/id',
         },
       },
+      body: {
+        '2xx': {
+          description: 'Update notePublicId',
+          properties: {
```

Comment:
seems wrong. It is not a `response` but `body`. Also, please add a response schema as well